vagga.yaml

```diff
@@ -51,6 +51,18 @@ containers:
       /run: !Tmpfs
         subdirs:
           postgresql: { mode: 0o777 }
+  test:
+    setup:
+      - !Ubuntu xenial
```

Comment:
'An unexpected error has occurred: CalledProcessError: Command: ('/root/.pre-commit/reponf39kcdj/py_env-default/bin/python3.5', '/root/.pre-commit/reponf39kcdj/py_env-default/bin/pip', 'install', '.')
Return code: 2
Expected return code: 0
'
Huge list of [Errno 95]. Then command failed
I've noticed before that some commands do not run under all OS